site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -1917,6 +1926,31 @@ public function showGrading(
         $this->core->getOutput()->renderOutput(['grading', 'ElectronicGrader'], 'popupSettings');
     }
 
+    /**
+     * Given a list of peers to grade, figure out how many teams we are
+     * actually grading.
+     *
+     * @param array<string> $people_assigned People we are being grading
+     * @param array<Team> $teams All teams for this gradeable.
+     * @return The number of teams this peer is grading.
```

Comment:
The "The" after the `@return` PHPDoc annotation is being interpreted as a type by PHPStan, and is causing a PHPStan error.